.github/workflows/plugin-cd.yml

```diff
@@ -0,0 +1,72 @@
+on:
+  workflow_call:
+    inputs:
+      artifacts:
+        default: dist/*.tar.gz
+        description: |
+          The path or files of artifacts to upload
+        required: false
+        type: string
+
+      bucket:
+        default: mattermost-plugins-ci/release/
+        description: |
+          The S3 bucket full path to upload
+        required: false
+        type: string
+
```

Comment:
Personal preference,  I would prefer loosing the NL here, for readability

Reply:
on this section I like it which makes me easier to follow but overall 0/5 too.